docroot/profiles/custom/cgov_site/modules/custom/cgov_yaml_content/cgov_yaml_content.module

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * @file
+ * Contains cgov_yaml_content.module.
+ */
+
+use \Drupal\Core\Entity\EntityInterface;
+
+/*
+ * This is going to be a fairly short treatise on the limitations of yaml_content module's
+ * event system. Basically the events like FIELD_IMPORT would lead to believe that you
+ * can manipulate the data coming in (getContentData), however, you cannot as the content
+ * data is immutable. After putzing for a while to find ways for us to replace the
+ * <drupal-entity> UUID attributes with the correctly loaded UUID, there appeared to be
+ * no sane way to accomplish this. (The yaml_content authors agreed it is a known limitation)
+ * SO... What we are going to do is to write some presave hooks that fire only when we
+ * are running the yaml_content module. (Oh, I should note that Paragraph entities got
+ * insanely complex to even attempt to fix pre-save as the entity revision was already
+ * saved at the point of pre-save for the node)
+ */
+
+/**
+ * Implements hook_entity_presave().
+ */
+function cgov_yaml_content_entity_presave(EntityInterface $entity) {
+  // Are we running from the command line?
+  if (PHP_SAPI !== 'cli') {
```

Comment:
@bryanpizzillo How do the PDQ imports run? Is there a performance concern, presuming every entity will make it to `$entity_embed_processor->processEntity($entity);`

Reply:
PDQ import run in the RESTApi, so that will not be `cli`.
But yeah, every entity will make it to the processor. I do not trust the hook_HOOK_presave hooks as I have experienced them getting missed. I REALLY did not like this approach, but there was little that could be done with `yaml_content` without rewriting it. In the Drupal slack channel, the maintainers acknowledge that updating the content is a gap...

Reply:
Ok, so I am just going to uninstall cgov_yaml_content after installation on the ODEs. It will stay on for local installs, since, well how else can you test content...